FFXIVClientStructs/FFXIV/Client/Game/Event/FishingEventHandler.cs

```diff
@@ -9,12 +9,73 @@ namespace FFXIVClientStructs.FFXIV.Client.Game.Event;
 [Inherits<EventHandler>, Inherits<AtkModuleInterface.AtkEventInterface>]
 [StructLayout(LayoutKind.Explicit, Size = 0x290)]
 public unsafe partial struct FishingEventHandler {
-    [FieldOffset(0x228)] public FishingState State;
-    [FieldOffset(0x230)] public bool CanFish;
-    [FieldOffset(0x23C)] public byte CurrentSelectedSwimBait;
-    [FieldOffset(0x240)] public uint SwimBaitId1;
-    [FieldOffset(0x244)] public uint SwimBaitId2;
-    [FieldOffset(0x248)] public uint SwimBaitId3;
+    [FieldOffset(0x220)] private byte Unk_220;
+    [FieldOffset(0x228)] public FishingState FishingState;
```

Comment:
```suggestion
    [FieldOffset(0x228)] public FishingState State;
```

Unnecessary rename of field.